data-prepper-logstash-configuration/src/main/resources/org/opensearch/dataprepper/logstash/mapping/kv.mapping.yaml

```diff
@@ -0,0 +1,11 @@
+pluginName: key_value
+mappedAttributeNames:
+  source: source
+  target: destination
+  field_split: field_delimiter_regex
```

Comment:
Are we supporting the following special chars provided in field_split with an exception? (`.`, `+`, `*`, `?`, `^`, `$`, `(`, `)`, `[`, `]`, `{`, `}`, `|`, `\`). I am concerned this will lead to a bad product experience using our converter and decreases our velocity of adoption. I believe a lot of potential customers like to split their data on `\n` and `|`. What can we do to improve this experience for them to migrate their field_split to field_delimiter_regex effectively?